mobile/lib/common/settings_screen.dart

```diff
@@ -58,15 +58,15 @@ class _SettingsScreenState extends State<SettingsScreen> {
       appBar: AppBar(title: const Text("Settings")),
       body: ScrollableSafeArea(
           child: Column(children: [
+        ElevatedButton(
+            onPressed: () {
+              rust.api.closeChannel();
```

Comment:
Imho we should not just expose `closeChannel` but do a smart check if there is still a DLC, if there is one, collab close the position first and then collab close the channel.

Reply:
In such a scenario I would prefer to instruct the user to close their position first.

Collab closing automatically is a bit weird because:

- The user likely wants to explicitly control their trades.
- We cannot even guarantee that the order will be executed any time soon.

Reply:
If we don't want this feature we should prevent him from clicking this button at least :)

Reply:
I like the idea with preventing the action when there's an open position.

Instead of disabling the button,  as it's hard for discoverability, I will display the snackbar saying "you have to close your position before closing the channel" and skip the footgun API call